agent/osv_agent.py

```diff
@@ -171,17 +173,17 @@ def _process_fingerprint_file(self, message: m.Message) -> None:
         path = message.data.get("path")
 
         if package_version is None:
-            logger.error("Error: Version must not be None.")
```

Comment:
shouldnt this be just warning or debug ?

Reply:
because it's none of those, it's not really a problem.